reference/src/client/test/sa_client_thread_test.cpp

```diff
@@ -33,8 +33,8 @@ sa_status client_thread_verify(
 
     sa_header header;
     sa_status status = sa_key_header(&header, key);
-    if (status != SA_STATUS_INVALID_PARAMETER) {
-        ERROR("key %d was found in a thread different from the one it was created in", key);
+    if (status != SA_STATUS_OK) {
+        ERROR("key %d was not found in a thread different from the one it was created in", key);
```

Comment:
Is the reference to thread accurate here or should it be application process?

Reply:
Yes it is correct. I reversed the test so it will try to retrieve the key/cipher/mac from a different thread in which it was create to make sure it can now be accessed. The test used to do the opposite it would make sure it couldn't find the key/cipher/mac in a different thread than the one in which it was created.